rasa/core/actions/forms.py

```diff
@@ -317,13 +317,26 @@ def extract_other_slots(
 
         return slot_values
 
+    def get_slot_to_fill(self, tracker: "DialogueStateTracker") -> Optional[str]:
+        """Get the slot to be filled.
+
+        When switching to another form, the requested slot setting is still from the
+        previous form and must be ignored.
+
+        Returns:
+        The slot name or None
+        """
+        return (
```

Comment:
could you please add a test which tests switching forms?

Reply:
Ok, let me try to figure out how to create a test that mimics the scenario of https://github.com/RasaHQ/rasa/issues/7710

Reply:
I added a test for switching forms. I commented it quite heavily. Let me know if that is OK or a bit too much comments.